ansible/roles/elastic.beats/.kitchen.yml

```diff
@@ -0,0 +1,126 @@
+---
```

Comment:
Why not just add this role to `requirements.yml` and use `ansible-galaxy install -r requirements.yml`?

Reply:
Is [this](https://github.com/dashevo/dash-network-deploy/pull/217/commits/2de94b846551bb81d1eaef3fb10fec8b38aa8455) what you mean?